torchaudio/functional.py

```diff
@@ -844,6 +844,147 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
 
 
 @torch.jit.script
+def gain(waveform, gain_db=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Apply amplification or attenuation to the audio signal.
+
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+       gain_db (float) Gain adjustment in decibels (dB) (Default: `1.0`).
+
+    Returns:
+       torch.Tensor: waveform amplified by gain_db
+    """
+    if (gain_db == 0):
+        return waveform
+
+    # gain_db = 20log(Tensor_output/tensor_input)
```

Comment:
Should this be removed?

Reply:
If gain_db is 0, shouldn't we return the original waveform? Otherwise the logic is `ratio = 0`, `waveform * 0`

Reply:
if `gain` is zero, the ratio is 1, and so it's fine to just return the waveform.

@cpuhrsch refers to the comment on line 861, no?